src/components/organisms/PaymentModal/PaymentModal.tsx

```diff
@@ -64,16 +56,7 @@ const PaymentModal: React.FunctionComponent<PropsType> = ({
       <PaymentConfirmation startUtcSeconds={selectedEvent.start_utc_seconds} />
     );
   } else if (!hasUserBoughtTicket) {
-    modalContent = (
-      <PaymentForm
-        setIsPaymentSuccess={() => setEventPaidSuccessfully(selectedEvent.id)}
-        setIsPaymentProceeding={setIsPaymentProceeding}
-        setIsCardBeingSaved={setIsCardBeingSaved}
-        isPaymentProceeding={isPaymentProceeding}
-        isCardBeingSaved={isCardBeingSaved}
-        event={selectedEvent}
-      />
-    );
+    modalContent = <div></div>;
```

Comment:
Why is this still here?

Reply:
fixed in 86ea8ad01c98485b02bc0029048517d83d6b9f3d